frontends/election-manager/src/screens/print_test_deck_screen.tsx

```diff
@@ -241,9 +245,61 @@ function useCreateHmpbTargetElements({
 const BmdPaperBallotsMemoized = React.memo(BmdPaperBallots);
 const HandMarkedPaperBallotsMemoized = React.memo(HandMarkedPaperBallots);
 
+interface PrintingModalProps {
+  advancePrinting: () => void;
+  currentPrecinct: Precinct;
+  precinctIds: string[];
+  printIndex: PrintIndex;
+}
+
+function PrintingModal({
+  advancePrinting,
+  currentPrecinct,
+  precinctIds,
+  printIndex,
+}: PrintingModalProps): JSX.Element {
+  if (printIndex.component === 'PaperChangeModal') {
+    return (
+      <Modal
+        centerContent
+        content={
+          <Prose textCenter>
+            <p>
+              Load printer with <b>legal-size paper</b>.
+            </p>
+          </Prose>
+        }
+        actions={
+          <Button onPress={advancePrinting}>
+            Legal Paper Loaded. Continue Printing
+          </Button>
```

Comment:
Lets make this button primary.
```suggestion
          <Button primary onPress={advancePrinting}>
            Legal Paper Loaded. Continue Printing
          </Button>
```